### ISSUE TITLE
Improve Accessibility Features of App

### ISSUE THREAD

**Issue by eddieTomo** [remihellotomo/Tomo-Office#105]
##  Summary

As the NHS, I want the Tomo App to be as accessible as possible, so users are not prevented from using it due to disability.

**DAQ Qs 7.15 - 7.15.2**
- Is your application a native iOS or Android app?
- What device OS accessibility features does your app use? e.g. VoiceOver (iOS), Dynamic Type (iOS), TalkBack (Android) or Select To Speak (Android)
- What steps have been taken to minimise battery usage?

##  Acceptance Criteria

- [ ] Consider how we might add in Accessibility Features to Tomo
  - [ ] Changeable font size
  - [ ] Colour blind mode
  - [ ] VoiceOver/TalkBack
  - [ ] Voice input methods.
- [ ] Write Issues and Implement desired features
- [ ] Document how we minimise battery usage

##  Definition of Done

- [ ] Accessibility features exist in the app
- [ ] Accessibility features are documented in the DAQ Responses
- [ ] Battery minimisation is documented in the DAQ Responses

**Comment by remihellotomo** [comment]
@eddieTomo - do you have a sense of how much Icons would improve our accessibility - aka for navigating the app, I want icons to accompany navigation, so we used to say Back but then I changed it to < Back, for Example.  Another is `About ?` versus just `About`. Issue ActivateMedia/tomo/issues/1061 is the epic of this.

**Comment by eddieTomo** [comment]
@remihellotomo I can't find guidance specifically from ORCHA on how icons fit into their UX/Accessiblity assessment however use of 'standard' icons do generally improve accessibility and ease of navigation due their broad familiarity and quick comprehension across languages.  Icons tend to be bigger than individual letters so can be easier to recognise by those with poor eyesight.  So yes I think we should aim to move to/include icons over time where possible

[NHS Digital guidelines](https://beta.nhs.uk/service-manual/styles-components-patterns/icons) make the point that whatever icons we used, they should always have the a high enough colour contract ratio of 4.5 to meet accessibility requirements
Good design practice guidelines such as [WCAG](https://www.w3.org/TR/WCAG21/) and Apple's [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines) which ORCHA **do** look for evidence of conformance to, state icons should have text alternatives - either alongside them, or in the code metadata for use by screenreading technology.
